game/game_ui.rb

```diff
@@ -0,0 +1,55 @@
+require_relative 'game'
+require_relative 'author'
+require_relative '../modules/game_storage'
+class GameUI
+  include SaveData
+  include LoadData
+  def initialize
+    @game_list = load_games
+    @author_list = load_authors
+  end
+
+  def create_game
+    puts 'Please enter the name of the game:'
+    game_name = gets.chomp
+    puts 'Please enter the last played date of the game:'
+    game_last_played_at = gets.chomp
+    puts 'Please enter the publish date for the game:'
+    game_publish_date = gets.chomp
+    puts 'Is it a multiplayer game? [y/n]'
+    game_multiplayer = gets.chomp.downcase == 'y'
+    @game_list.push Game.new(game_name, game_last_played_at, game_publish_date, game_multiplayer)
+    save_game(@game_list)
+    puts 'Please enter the author\'s first name and the last name of the game (e.g. “Stephen King”)'
+    first_name = gets.chomp
+    last_name = gets.chomp
+    @author_list.push Author.new(first_name, last_name)
+    save_author(@author_list)
+    puts "\nNew game created successfully!\n"
+  end
```

Comment:
- [ ] Currently, I am having this error while creating a game so make sure that you fix this error and please ensure that the app works perfectly without any errors :+1: 
  
  ![image](https://github.com/QueenterJuma/ruby_catalog/assets/92303891/84e864ca-a0f1-4312-9707-e423133a722a)

Reply:
Done. Thanks